momento-sdk/src/main/java/momento/sdk/messages/CacheDictionaryFetchResponse.java

```diff
@@ -54,6 +54,19 @@ public Map<String, String> valueDictionaryStringString() {
                   entry -> entry.getValue().toStringUtf8()));
     }
 
+    /**
+     * Gets the retrieved values as a dictionary of UTF-8 string keys and values.
+     *
+     * @return the dictionary.
+     */
+    public Map<String, String> valueDictionary() {
+      return byteStringKeysValues.entrySet().stream()
```

Comment:
Can this just call `valueDictionaryStringString()`?

Reply:
That function exists. This is just an alias of that function